wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+Files.swift

```diff
@@ -99,19 +99,8 @@ extension ConversationInputBarViewController {
 
                 self.impactFeedbackGenerator.impactOccurred()
 
-                var conversationMediaAction: ConversationMediaAction = .fileTransfer
-
                 do {
                     let message = try conversation.appendFile(with: metadata)
-                    if let fileMessageData = message.fileMessageData {
-                        if fileMessageData.isVideo {
-                            conversationMediaAction = .videoMessage
-                        } else if fileMessageData.isAudio {
-                            conversationMediaAction = .audioMessage
-                        }
-                    }
-
-                    Analytics.shared.tagMediaActionCompleted(conversationMediaAction, inConversation: conversation)
```

Comment:
If we no longer use `message`, then we don't need to store it.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/57faee4725c8ea6660c3da1bbc75e3f4629f3f9b